Neos.Flow/Classes/Error/DebugExceptionHandler.php

```diff
@@ -137,7 +135,7 @@ protected function renderStatically(int $statusCode, \Throwable $exception)
         if (Bootstrap::$staticObjectManager instanceof ObjectManagerInterface) {
             $bootstrap = Bootstrap::$staticObjectManager->get(Bootstrap::class);
             $footer .= '<tr><th>Application Context</th><td class="ExceptionProperty">' . $bootstrap->getContext() . '</td></tr>';
-            $footer .= '<tr><th>Request Handler</th><td class="ExceptionProperty">' . get_class($bootstrap->getActiveRequestHandler()) . '</td></tr>';
+            $footer .= '<tr><th>Request Handler</th><td class="ExceptionProperty">' . $bootstrap->getActiveRequestHandler()::class . '</td></tr>';
```

Comment:
Looks weird but guess it's valid syntax.

Reply:
Ha I thought the same